app/scripts/controllers/dashboard.coffee

```diff
@@ -326,8 +326,15 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', [
           loadingSrv.containerLoad schoolsP, document.getElementById mapId
           schoolsP.then (schools) ->
             if $scope.selected? and $scope.viewMode == 'schools'
+              $scope.selectedCodeYear =
```

Comment:
Still looking at this commit to understand what the problem was, but,
We should try to only write to variables on `$scope` with a `watchCompute`